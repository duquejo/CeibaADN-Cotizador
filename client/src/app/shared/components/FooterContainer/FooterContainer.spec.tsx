import * as React from 'react';
import { ShallowWrapper, mount, render, shallow } from 'enzyme';
import { FooterContainer } from './index';


describe('Pruebas sobre componente <FooterContainer />', () => {

    let componentWrapper: ShallowWrapper;

    beforeEach( () => {
        componentWrapper = shallow(<FooterContainer/>);
    });

    it('DeberÃ­a renderizar el elemento', () => {
        expect( componentWrapper ).toMatchSnapshot();
    });
});