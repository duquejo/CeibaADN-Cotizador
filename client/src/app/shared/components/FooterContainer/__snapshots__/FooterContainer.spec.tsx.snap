// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre componente <FooterContainer /> Debería renderizar el elemento 1`] = `
<styled.footer>
  José Miguel Duque Mejía - 
  2021
</styled.footer>
`;
